src/Uno.UI.Maps/MapPresenter.Android.cs

```diff
@@ -144,7 +144,7 @@ private void HandleActivityLifeCycle()
 		{
 			_callbacks = new MapLifeCycleCallBacks(onPause: _internalMapView.OnPause, onResume: _internalMapView.OnResume);
 
-			_application = Context.ApplicationContext as Android.App.Application;
+			_application = (Context!.ApplicationContext as Android.App.Application)!;
```

Comment:
We're already checking as after, why not `Context?.` ?

Reply:
```suggestion
			_application = (Context?.ApplicationContext as Android.App.Application)!;
```